admin-base/ui.apps/src/main/js/perAdminApp.js

```diff
@@ -425,17 +439,22 @@ function findActionInTree(component, command, target) {
  * @param target
  */
 function actionImpl(component, command, target) {
-    if(component.$options.methods && component.$options.methods[command]) {
-        return component.$options.methods[command](component, target)
+  if (component.$options.methods && component.$options.methods[command]) {
+    return component.$options.methods[command](component, target);
+  } else {
+    if (component.$parent === component.$root) {
+      if (!findActionInTree(component.$root, command, target)) {
+        logger.error(
+          'action',
+          command,
+          'not found, ignored, traget was',
+          target
+        );
+      }
     } else {
-        if(component.$parent === component.$root) {
-            if(!findActionInTree(component.$root, command, target)) {
-                logger.error('action', command, 'not found, ignored, traget was', target)
```

Comment:
I know this is probably just reformatted, but:
* target instead of traget
* could we use templated strings?

Reply:
typoo fixed. there is advantages of using `...args` syntax here as the object is not transfered to a string in console but rather displayed indepth